packages/bits/src/lib/wizard-v2/wizard-step-header/wizard-step-header.component.less

```diff
@@ -1,63 +1,53 @@
 @import "../wizard-vars";
 @import (reference) "../../../styles/nui-framework-variables.less";
 @import (reference) "../../../styles/mixins.less";
+@import (reference) "../../../styles/nui-framework-typography.less";
 
-.nui-wizard-step {
+.nui-wizard-step-header {
+    overflow: hidden;
+    outline: none;
+    cursor: pointer;
+    position: relative;
+    .setCssVariable(color, nui-color-text-default);
+    width: @wizard-label-header-width;
 
-    &-header {
-        overflow: hidden;
-        outline: none;
-        cursor: pointer;
-        position: relative;
-        box-sizing: content-box;
-        -webkit-tap-highlight-color: transparent;
-        .setCssVariable(color, nui-color-text-default);
-        width: @wizard-label-header-width;
-
-        &--completed {
-            .setCssVariable(color, nui-color-text-link);
-
-            .nui-wizard-horizontal-line {
-                border: 10px red solid !important;
-                .setCssVariable(border-color, nui-color-text-link);
-            }
-        }
+    &--completed {
+        .setCssVariable(color, nui-color-text-link);
+    }
 
-        &--disabled {
-            pointer-events: none;
-            cursor: default;
-        }
+    &--disabled {
+        pointer-events: none;
+        cursor: default;
+    }
 
-        .nui-step-header-ripple {
-            pointer-events: none;
-        }
+    .nui-step-header__ripple {
+        pointer-events: none;
+    }
 
-        &:focus {
-            // TODO: switch back to mixin .focus-outline() when z-index is removed in the scope of NUI-6103
-            outline: none;            
-            box-shadow: inset 0 0 0 @nui-line-medium fade(@nui-color-selected-contrast, 50%);
-            border-radius: @nui-radius-default;
-        }
+    &:focus {
+        // TODO: switch back to mixin .focus-outline() when z-index is removed in the scope of NUI-6103
+        outline: none;
+        box-shadow: inset 0 0 0 @nui-line-medium
+            fade(@nui-color-selected-contrast, 50%);
```

Comment:
maybe keeping in inline would look better? what do you think?

Reply:
Will fix, thanks